res/qml/main.qml

```diff
@@ -64,6 +65,17 @@ Rectangle {
                     checkable: true
                 }
 
+                Skin.Button {
+                    id: showPreferencesButton
+
+                    text: "Prefs"
+                    activeColor: Theme.white
+                    checkable: true
+                    onClicked: {
+                        Mixxx.PreferencesDialog.show();
+                    }
```

Comment:
Not sure if it makes sense for the button to be checkable, when nothing happens when unchecking.

Reply:
I tried to add a Q_PROPERTY to the QmlDlgPreferencesProxy class indicating the state of the dialog but unfortunately there is no notifyChanged callback for QWidget's `visible` property.

Reply:
Simply removing `checkable: true` works well.